.travis.yml

```diff
@@ -1,14 +1,15 @@
+sudo: false
 language: node_js
 before_install:
   - curl --location http://git.io/1OcIZA | bash -s
```

Comment:
You're talking about [L4](https://github.com/rvagg/bl/pull/21#discussion-diff-32892797R4)? That's not related to this PR but it's OK to remove it.